zipkin-server/src/test/java/zipkin2/server/internal/prometheus/ITZipkinMetrics.java

```diff
@@ -200,11 +201,14 @@ private Response post(String path, byte[] body) throws IOException {
       .build()).execute();
   }
 
-  static List readJson(String json) throws Exception {
-    ObjectMapper mapper = new ObjectMapper();
-    JsonNode jsonNode = mapper.readTree(json);
-    List<String> fieldsList = new ArrayList<>();
-    jsonNode.fieldNames().forEachRemaining(fieldsList::add);
-    return fieldsList;
+  static Map<String, Integer> readJson(String json) throws Exception {
+    Map<String, Integer> result = new LinkedHashMap<>();
```

Comment:
Maybe it's fine to parse straight into a map for tests

Reply:
I'll leave this as plan to remove databind and prefer to not re-introduce it just to remove it